docs/typescript/how-to-develop-a-workflow-definition-in-typescript.md

```diff
@@ -0,0 +1,37 @@
+---
+id: how-to-develop-a-workflow-definition-in-typescript
+title: How to develop a Workflow definition in Typescript
+sidebar_label: Develop a Workflow definition
+description: Develop a Workflow definition
+tags:
+  - developer-guide
+  - sdk
+  - typescript
+---
+
+A Workflow Function has two parts:
+
+- The function name is known as the Workflow Type.
+- The function implementation code (body) is known as the Workflow Definition.
+
+Each Workflow Definition is bundled with any third party dependencies, and registered by Workflow Type in a Worker. A Workflow function becomes a Workflow Execution (instance) only when started from a Workflow Client using its Workflow Type.
+
+Workflow Definitions are _just functions_, which can store state, and orchestrate Activity Functions.
+The following code snippet uses `proxyActivities` to create functions to schedule a `greet` Activity in the system to say hello.
+
+A Workflow Function can have multiple parameters, however, Temporal encourages you to use a single object parameter.
```

Comment:
```suggestion
A Workflow Function can have multiple parameters; however, we recommend using a single object parameter.
```

"Temporal encourages" makes me think something about the system encourages it, and makes me wonder what part of the system/how.

Reply:
Move to below snippet?